### ISSUE TITLE
Crashes on publish

### ISSUE THREAD

**Issue by rossjones** [theodi/octopub#859]
Created a test dataset, chose to publish it at `rossjones` organization (my personal account). It crashed ... "We're sorry, something went wrong".

https://octopub.io/collection/7/publish_dataset

### Expected Behaviour
It should be published

### Desired Behaviour (for improvement suggestions only)
It should not crash

### Current Behaviour (for problems)

1. It crashes
2. It shows a pointless error page which doesn't help me, or suggest how to report the issue
3. It DID actually publish the collection - https://octopub.io/collection/7

### Your Environment

FF on Desktop

**Comment by peterkwells** [comment]
Ditto with https://octopub.io/collection/10

**My environment**

Chrome on desktop

**Comment by olivierthereaux** [comment]
@rossjones can you point to the dataset as published? 

Dataset pages on Octopub are not public any more so I can not see the pages you link to.

**Comment by rossjones** [comment]
Ah I'm sorry, discomfort at the permissions requested has made me remove the app's permissions from my github account, so I can't replicate.  Perhaps I was mistaken but I recall on the dashboard it suggested it was published - I could be wrong.  Presumably @peterkwells can replicate though.

**Comment by olivierthereaux** [comment]
You're not the first to find the broad requested permissions rather aggressive @rossjones . There were earlier discussions on this, e.g. #187. Thanks for reporting, I'm sure we'll find a way to reproduce.

**Comment by peterkwells** [comment]
same issue with a new dataset. Octopub ID is https://octopub.io/collection/12.

I created a new collection, uploaded file, navigated to data collection, clicked publish and got a fatal - and unhelpful - error.

The actual dataset was published after checking with a manual navigation to the github pages version, see: https://peterkwells.github.io/a-list-of-musicians-and-bands-that-cause-harmful-impact/

**Comment by peterkwells** [comment]
just tried again, still a problem. Resulting github repo: https://github.com/peterkwells/things-about-trees

**Comment by peterkwells** [comment]
it crashed again
<img width="864" alt="screen shot 2018-09-26 at 12 34 28" src="https://user-images.githubusercontent.com/6670237/46077374-8c65e080-c188-11e8-83cf-54f3b9844295.png">

**Comment by langphil** [comment]
Hi @peterkwells, can you send over the files that you have been trying to publish?

**Comment by peterkwells** [comment]
as zip as github wouldn't let me upload a csv

[bands and their favourite CRMs - Sheet1.csv.zip](https://github.com/theodi/octopub/files/2419504/bands.and.their.favourite.CRMs.-.Sheet1.csv.zip)

**Comment by langphil** [comment]
Are you seeing this warning when attempting to upload. The issue is with the naming of the file you are attempting to upload.

Changing the filename and attempting to upload has shown success.

![screenshot 2018-09-26 at 12 53 47](https://user-images.githubusercontent.com/5029907/46078319-ad7c0080-c18b-11e8-9a10-7b894d6e8ddb.png)

**Comment by caiwilliamson** [comment]
I tried uploading and publishing "bands and their favourite CRMs - Sheet1.csv" with no issues. File naming does not appear to be the issue since upload is prevented if the file name fails validation. Closing this as there is not enough information to replicate the issue with any degree of accuracy. If problems like this occur in the future, request the time the user attempted to upload/publish and check the production application logs on Heroku to diagnose the precise issue.
